### PR TITLE
Bump rubygem-mustermann to 1.1.1

### PR DESCRIPTION
**Pull request by ekohl**
This version contains fixes for warnings generated by Ruby 2.7. Those could be seen in the Smart Proxy on startup.

It also adds a dependency on rubygem-ruby2_keywords which is also added.

### CODE REVIEW

File: packages/foreman/rubygem-ruby2_keywords/rubygem-ruby2_keywords.spec [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name ruby2_keywords
+%global gem_require_name %{gem_name}
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.0.4
+Release: 1%{?dist}
+Summary: Shim library for Module#ruby2_keywords
```
Comment (evgeni) [inline packages/foreman/rubygem-ruby2_keywords/rubygem-ruby2_keywords.spec:11]: I have a slight hunch that this won't render nicely ;)

Comment (ekohl) [reply packages/foreman/rubygem-ruby2_keywords/rubygem-ruby2_keywords.spec]: Updated

File: packages/foreman/rubygem-ruby2_keywords/rubygem-ruby2_keywords.spec [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}ruby
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+# end specfile generated dependencies
+
+%description
+Shim library for Module#ruby2_keywords.
```
Comment (evgeni) [inline packages/foreman/rubygem-ruby2_keywords/rubygem-ruby2_keywords.spec:29]:
```suggestion
Shim library for Module ruby2_keywords.
```

**Review by evgeni — APPROVED** [review]
I'm picky, but I ACK :)

**Comment by ekohl** [comment]
Updated

**Review by ehelms — APPROVED** [review]

**Pull request merged**